vllm/model_executor/models/minicpmv.py

```diff
@@ -23,18 +23,30 @@
 """Inference-only MiniCPM-V-2 model compatible with HuggingFace weights."""
```

Comment:
Please add type annotations to the file, by the way.

Reply:
Is it ok with
```
"""Inference-only MiniCPM-V model compatible with HuggingFace weights."""
```

Reply:
By type annotations, I mean [this](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#functions).

Reply:
> By type annotations, I mean [this](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#functions).

Forget this..plz wait